### DISCUSSION TITLE
Student Developer Pack

### DISCUSSION THREAD

**Discussion by Aurokrishnaa** [community/community#147986] [GitHub Education]

**Comment by jgcamiloaga** [comment]
In my case, I was able to use my phone to take a photo of my academic progress, and a few days later, my application was accepted. I think the issue might be that the photo doesn’t show a date that clearly indicates you are currently studying.

![Captura de pantalla 2024-12-27 114419](https://github.com/user-attachments/assets/5bdc084e-b610-4164-b60a-ef2da3b241f2)

**Reply by Spaun12** [reply]
I have tried so many times myself. It is very weird that they give you no recourse. Not very supportive.  Or maybe people aren't important anymore due to AI.
